2021-11-11 22:12:18.936 | INFO     | httprunner.runner:test_start:450 - Start to run testcase: case003, TestCase ID: 27d12edc-eadb-4905-a7f3-f4a39a03b7c8
2021-11-11 22:12:18.937 | INFO     | httprunner.runner:__run_step:292 - run step begin: step_name >>>>>>
2021-11-11 22:12:22.962 | WARNING  | httprunner.client:request:188 - failed to get client address info: 'NoneType' object has no attribute 'connection'
2021-11-11 22:12:22.962 | WARNING  | httprunner.client:request:196 - failed to get server address info: 'NoneType' object has no attribute 'connection'
2021-11-11 22:12:22.963 | DEBUG    | httprunner.client:log_print:40 - 
================== request details ==================
method   : GET
url      : http://localhost/demo/path
headers  : {}
cookies  : {}
body     : None

2021-11-11 22:12:22.968 | DEBUG    | httprunner.client:log_print:40 - 
================== response details ==================
status_code : 0
headers  : {}
cookies  : {}
encoding : None
content_type : 
body     : 

2021-11-11 22:12:22.968 | ERROR    | httprunner.client:request:215 - HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /demo/path (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001157F04EF40>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))
2021-11-11 22:12:22.969 | INFO     | httprunner.runner:__run_step:304 - run step end: step_name <<<<<<

